core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3911,8 +3911,8 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.SERVER)
           .build();
-  public static final PropertyKey PROXY_S3_COMPLETE_MULTIPART_UPLOAD_KEEPALIVE_TIME =
-      new Builder(Name.PROXY_S3_COMPLETE_MULTIPART_UPLOAD_KEEPALIVE_TIME)
+  public static final PropertyKey PROXY_S3_COMPLETE_MULTIPART_UPLOAD_KEEPALIVE_TIME_INTERVAL =
```

Comment:
@HelloHorizon Were you thinking of something like `HEARTBEAT` instead of `KEEPALIVE`? I didn't find any specific choice of terminology in the S3 docs. I'm fine with either.

Reply:
I am fine with both